src/base/bittorrent/session.cpp

```diff
@@ -1710,7 +1720,12 @@ void Session::handleTorrentFinished(TorrentHandle *const torrent)
 void Session::handleTorrentResumeDataReady(TorrentHandle *const torrent, const libtorrent::entry &data)
 {
     --m_numResumeData;
-    writeResumeDataFile(torrent, data);
+
+    QByteArray out;
+    libt::bencode(std::back_inserter(out), data);
```

Comment:
OK. Especially the 2nd one makes sense. It would be good to leave a small comment here indicating the rationale to the future reader.